localization/english/event_localization/wc_major_decisions_events_l_english.yml

```diff
@@ -0,0 +1,41 @@
+ wc_holy_order_decision.0011.desc.original_paladin:0 "[founder.GetTitledFirstName] has sent word to all priests of [founder.GetFaith.HighGodName]. War is coming, and the faithful must be ready.\n\nIf humanity and its allies are to survive the coming era of strife, it will need warriors who can heal the innocent and strike down the wicked.\n\nThese warriors will be called "paladins". Among the faithful, five men, including myself, have been chosen as the inaugural warriors. Renowned for our strength and devotion, a libram of power has been bestowed upon each of us, entrusting us with the duty to train those who will follow. Since ancient times, it is said that humanity had a protector, the mythical hero Tyr. Our order of paladins will be known as the Knights of [leader.GetPrimaryTitle.GetBaseName], in honor of this legendary figure."
+ wc_holy_order_decision.0011.desc.original_paladin_northshire_stay:0 "[founder.GetTitledFirstName] has sent word to all priests of [founder.GetFaith.HighGodName]. War is coming, and the faithful must be ready.\n\nIf humanity and its allies are to survive the coming era of strife, it will need warriors who can heal the innocent and strike down the wicked.\n\nThese warriors will be called "paladins". Among the faithful, five men, including myself, have been chosen as the inaugural warriors. Renowned for our strength and devotion, a libram of power has been bestowed upon each of us, entrusting us with the duty to train those who will follow. Since ancient times, it is said that humanity had a protector, the mythical hero Tyr. Our order of paladins will be known as the Knights of [leader.GetPrimaryTitle.GetBaseName], in honor of this legendary figure."
+ wc_holy_order_decision.0011.a:0 "Gloria, Honorifice, Praelium Facio!"
+ wc_holy_order_decision.0011.a_other:0 "A worrying development..."
+
+ wc_holy_order_decision.0012:0 "We both know that a darkness is seeping into the land. Our allies report skirmishes everywhere I am sending you this message because I know you will not stand idly by as the forces of hell bring ruin to the world. We will need you, and people like you, in the coming wars. I want you to be among the first to take up our cause. Should you wish to walk this path, join us. There is much to discuss."
```

Comment:
```suggestion
 wc_holy_order_decision.0012:0 "We both know that a darkness is seeping into the land. Our allies report skirmishes everywhere. I am sending you this message because I know you will not stand idly by as the forces of hell bring ruin to the world. We will need you, and people like you, in the coming wars. I want you to be among the first to take up our cause. Should you wish to walk this path, join us. There is much to discuss."
```